pytorch_fid/fid_score.py

```diff
@@ -96,32 +103,27 @@ def get_activations(files, model, batch_size=50, dims=2048, cuda=False):
         print(('Warning: batch size is bigger than the data size. '
                'Setting batch size to data size'))
         batch_size = len(files)
+         
+    ds = ImagesPathDataset(files, transforms=TF.ToTensor())
+    dl = torch.utils.data.DataLoader(ds, batch_size=batch_size, drop_last=True, num_workers=cpu_count())
```

Comment:
This should be `drop_last=False`, otherwise `pred_array` might not be completely filled.

Reply:
Done